controllers/hostedcontrolplane/hostedcontrolplane.go

```diff
@@ -87,9 +100,24 @@ func (r *HostedControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.R
 		return utilreconcile.RequeueWith(err)
 	}
 
+	//Create Dynatrace API client
+	valueDynatraceApiToken, valueDynatraceTenant, err := r.getDynatraceSecrets(ctx)
+	if err != nil {
+		log.Info("Error getting secret")
+		return utilreconcile.RequeueWith(err)
+	}
+	baseURL := fmt.Sprintf("%s/v1", valueDynatraceTenant)
+	DynatraceAPIClient := dynatrace.NewDynatraceAPIClient(baseURL, valueDynatraceApiToken)
```

Comment:
```suggestion
	dynatraceAPIClient := dynatrace.NewDynatraceAPIClient(baseURL, valueDynatraceApiToken)
```
This happens a couple times in this PR and I'll just comment on this one, but lets keep variable names camelCase.

Reply:
Updated :)